### DISCUSSION TITLE
Wazuh 4.0 Released

### DISCUSSION THREAD

**Discussion by greatapoc** [Security-Onion-Solutions/securityonion#1746] [Ideas] [answered]
Hi guys, no doubt you'll get to it eventually just thought I'd point out that Wazuh 4.0 has been released. Looks like there's change related to enrollment which might make things a bit easier for people. Maybe a custom agent package could be rolled up with enrollments built into it.

https://documentation.wazuh.com/4.0/release-notes/release_4_0_0.html

**Comment by weslambert** [comment, marked as answer]
I'll add this to the roadmap.  Thanks!

On Fri, Oct 30, 2020 at 12:28 AM greatapoc <notifications@github.com> wrote:

> Hi guys, no doubt you'll get to it eventually just thought I'd point out
> that Wazuh 4.0 has been released. Looks like there's change related to
> enrollment which might make things a bit easier for people. Maybe a custom
> agent package could be rolled up with enrollments built into it.
>
> https://documentation.wazuh.com/4.0/release-notes/release_4_0_0.html
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Security-Onion-Solutions/securityonion/discussions/1746>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AEAM3KGQ7C7G7FMUILBORBLSNI6GXANCNFSM4TERBJWA>
> .
>


-- 
https://twitter.com/therealwlambert
https://securityonion.net/
